ydb/core/tx/datashard/export_s3_uploader.cpp

```diff
@@ -812,6 +926,28 @@ IActor* TS3Export::CreateUploader(const TActorId& dataShard, ui64 txId) const {
         ? GenYdbScheme(Columns, Task.GetTable())
         : Nothing();
 
+    const auto& persQueuesTPathDesc = Task.GetChangefeedUnderlyingTopics();
+    const auto& cdcStreams = Task.GetTable().GetTable().GetCdcStreams();
+    Y_ASSERT(persQueuesTPathDesc.size() == cdcStreams.size());
+
+    const int changefeedsCount = cdcStreams.size();
+    TVector <TChangefeedExportDescriptions> changefeedsExportDescs;
+    changefeedsExportDescs.reserve(changefeedsCount);
+
+    for (int i = 0; i < changefeedsCount; ++i) {
+        Ydb::Table::ChangefeedDescription changefeedDesc;
+        Ydb::Topic::DescribeTopicResult topic;
+        const auto& cdcStream = cdcStreams[i];
+        const auto& persQueueTPathDesc = persQueuesTPathDesc[i];
+        FillChangefeedDescription(changefeedDesc, cdcStream);
+        Ydb::StatusIds_StatusCode status;
+        TString error;
+        FillTopicDescription(topic, persQueueTPathDesc.GetPersQueueGroup(),
+        persQueueTPathDesc.GetSelf(),
+        cdcStream.GetName(), status, error);
+        changefeedsExportDescs.emplace_back(changefeedDesc, topic);
```

Comment:
```suggestion
        Ydb::Table::ChangefeedDescription changefeed;
        const auto& cdcStream = cdcStreams.at(i);
        FillChangefeedDescription(changefeed, cdcStream);

        Ydb::Topic::DescribeTopicResult topic;
        const auto& pq = persQueuesTPathDesc.at(i);
        Ydb::StatusIds::StatusCode status;
        TString error;
        FillTopicDescription(topic, pq.GetPersQueueGroup(), pq.GetSelf(), cdcStream.GetName(), status, error);

        changefeedsExportDescs.emplace_back(changefeed, topic);
```

Общий совет: объявлять переменные поближе к месту использования. Еще я предлагаю немного сократить названия переменных.